ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -39,3 +39,8 @@ def pinv(a, rcond=None, validate_args=False, name=None):
 
 
 pinv.unsupported_dtypes = ("float16", "bfloat16")
+
+def tensordot(x,y,axes,name=None):
+    return ivy.tensordot(x,y,axes)
+
+tensordot.unsupported_dtypes = ("float16", "bfloat16")
```

Comment:
Tensordot() for tensorflow supports only `float32` and `float64` according to the docs (https://www.tensorflow.org/api_docs/python/tf/tensordot).
Kindly add only the supported_dtypes.